packages/jellyfish-api-core/src/category/account.ts

```diff
@@ -311,6 +311,15 @@ export class Account {
   ): Promise<number> {
     return await this.client.call('accounthistorycount', [owner, options], 'number')
   }
+
+  /**
+   * Returns information about current anchor bonus, incentive funding, burnt token(s)
+   *
+   * @return {Promise<CommunityBalanceData>}
+   */
+  async listCommunityBalances (): Promise<CommunityBalanceData> {
+    return await this.client.call('listcommunitybalances', [], 'number')
```

Comment:
To prevent precision lost of floating point, we need use `bignumber`
```
listcommunitybalances
{
  "AnchorReward": 41.638112,
  "IncentiveFunding": 37564.5671231,
  "Burnt": 2568045.098536
}
```